terraform/roles.tf

```diff
@@ -288,3 +288,58 @@ resource aws_iam_role_policy_attachment ec2_readonly {
   role       = aws_iam_role.caascad_operator.name
   policy_arn = "arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess"
 }
+
+data aws_iam_policy_document caascad_monitoring_cloud_services_assumerole_policy {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "AWS"
+      identifiers = [var.caascad_operator_trusted_arn]
+    }
+  }
+}
+
+data aws_iam_policy_document caascad_cloudwatch_metrics_readonly_policy {
+  statement {
+    actions = [
+      "tag:GetResources",
+      "cloudwatch:ListTagsForResource",
+      "cloudwatch:GetMetricStatistics",
+      "cloudwatch:GetMetricData",
+      "cloudwatch:ListMetrics"
+    ]
+    resources = [
+      "*"
+    ]
+  }
+}
+
+resource aws_iam_policy caascad_cloudwatch_metrics_readonly_policy {
+  name = "caascad-cloudwatch-metrics-readonly"
+  path = "/"
+
+  policy = data.aws_iam_policy_document.caascad_cloudwatch_metrics_readonly_policy.json
+}
+
+resource aws_iam_role caascad_monitoring_cloud_services {
+  name        = "caascad-monitoring-cloud-services"
+  description = "Role used for Caascad monitoring of cloud services"
+
+  force_detach_policies = true
+
+  max_session_duration = 10 * 60 * 60 // 10 hours
```

Comment:
I don't know which duration would be best. Minimum is 1 hour.

Reply:
@mvienne-orange : set 1 hour to see potential refresh issues quickly